packages/fxa-auth-server/lib/account-delete.ts

```diff
@@ -47,25 +56,38 @@ export class AccountDeleteManager {
   private playBilling?: PlayBilling;
   private log: Log;
   private config: ConfigType;
+  private glean: GleanMetricsType;
+  private statsd: StatsD;
+  private inactiveAccountsManager: InactiveAccountsManager;
 
   constructor({
     fxaDb,
     oauthDb,
     config,
     push,
     pushbox,
+    mailer,
+    statsd,
+    glean,
+    log,
   }: {
     fxaDb: DB;
     oauthDb: OAuthDbDeleteAccount;
     config: ConfigType;
     push: PushForDeleteAccount;
     pushbox: PushboxDeleteAccount;
+    mailer: any;
```

Comment:
Wonder if we should make a MailerType eventually.

Reply:
Definitely couldn't hurt.  And as part of that we could define a type for each of the email templates/types, and that type would require a DB id, so that we won't forget to add a migration to add the email type.